pyproject.toml

```diff
@@ -6,6 +6,7 @@
 requires = ["setuptools<74"]
 build-backend = "setuptools.build_meta"
 
+[tool.coverage]
```

Comment:
```suggestion
```

A newer version of `tomlkit` no longer generates empty sections. Your sandbox may be outdated.

Reply:
Thank you for noticing it. I updated my toolchain an the empty section is now gone.